packages/ember-glimmer/lib/template.js

```diff
@@ -2,6 +2,14 @@ import { Template } from 'glimmer-runtime';
 
 class Wrapper {
   constructor(id, env, spec) {
```

Comment:
The factory should be passed the owner, we should use the owner of the template, the env really should come from the app and be passed into the engine with instantiate false.  /cc @dgeb